tv/src/main/java/io/filmtime/tv/ui/theme/Theme.kt

```diff
@@ -0,0 +1,34 @@
+package io.filmtime.tv.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+import androidx.tv.material3.ExperimentalTvMaterial3Api
+import androidx.tv.material3.MaterialTheme
+import androidx.tv.material3.darkColorScheme
+import androidx.tv.material3.lightColorScheme
+
+@OptIn(ExperimentalTvMaterial3Api::class)
+@Composable
+fun FilmTimeTheme(
+  isInDarkTheme: Boolean = isSystemInDarkTheme(),
```

Comment:
Do we need light colors on TV?

Reply:
I'm not sure, but they were generated when I created a TV application module